Source/Core/AST/AbsyCmd.cs

```diff
@@ -4003,6 +4015,9 @@ protected override Cmd ComputeDesugaring(PrintOptions options)
         Contract.Assert(e != null);
         Expr copy = Substituter.ApplyReplacingOldExprs(calleeSubstitution, calleeSubstitutionOld, e.Condition);
```

Comment:
Could you follow-up this PR with one where many of the AST classes are moved to separate files? Here for example it's relatively difficult for me to see what `this` in `this.Proc.Ensures` points to, since `CallCmd` is not in its own file.

Reply:
I can probably do this in a couple of weeks, but certainly wouldn't object if you did it sooner! :)